articles/multifactor-authentication/factors/sms.md

```diff
@@ -25,10 +24,8 @@ Your users must have a device capable of receiving SMS messages to use this opti
 
 ## Administrative setup
 
-To set up SMS and be able to send SMS messages to users, you'll need to enable the SMS factor in the Dashboard and [configure Twilio](/multifactor-authentication/twilio-configuration). You can (optionally) [customize your SMS notification templates](/multifactor-authentication/sms-templates).
+To set up SMS and be able to send SMS messages to users, you'll need to enable the SMS factor in the Dashboard and [configure Twilio](/multifactor-authentication/twilio-configuration), or alternatively use your own [custom SMS gateway](multifactor-authentication/send-phone-message-hook-configuration) with the Send Phone Message feature. 
 
-::: note
-Custom SMS gateways are unavailable with MFA.
-:::
+You can (optionally) [customize your SMS notification templates](/multifactor-authentication/sms-templates).
 
 ![MFA SMS Settings](/media/articles/multifactor-authentication/sms-settings.png)
```

Comment:
We need to update the sms-settings screenshot

Reply:
Can I get the new one? Or at least the Figma Link?

Reply:
Nevermind I got it, will push it later

Reply:
Added it